lib/cm_admin/view_helpers/form_field_helper.rb

```diff
@@ -5,21 +5,21 @@ def input_field_for_column(f, field)
         value = field.custom_value || f.object.send(field.field_name)
         case field.input_type
         when :integer
-          return f.text_field field.field_name, class: 'normal-input', disabled: field.disabled, value: value, data: {behaviour: 'integer-only'}
+          return f.text_field field.field_name, class: 'normal-input', disabled: field.disabled, value: value, placeholder: "Enter #{field.field_name.to_s.downcase.gsub('_', ' ')}", data: { behaviour: 'integer-only' }
```

Comment:
@anbublacky - Can we use `humanize` instead of `gsub`?
```suggestion
          return f.text_field field.field_name, class: 'normal-input', disabled: field.disabled, value: value, placeholder: "Enter #{field.field_name.to_s.humanize}", data: { behaviour: 'integer-only' }
```

Reply:
@anbublacky - We can downcase after humanize?
```suggestion
          return f.text_field field.field_name, class: 'normal-input', disabled: field.disabled, value: value, placeholder: "Enter #{field.field_name.to_s.humanize.downcase}", data: { behaviour: 'integer-only' }
```